inverse_covariance/inverse_covariance.py

```diff
@@ -12,6 +13,9 @@ def _init_coefs(X, method='corrcoef'):
     elif method == 'cov':
```

Comment:
As an aside, did we ever figure out why sometimes one of these original initializers barfs?